src/model/cloud-runner/services/core/shared-workspace-locking.ts

```
@@ -1,23 +1,53 @@
+    }
+    const result = await SharedWorkspaceLocking.s3.send(
+      new ListObjectsV2Command({ Bucket: bucket, Prefix: prefix, Delimiter: '/' }),
+    );
+    const entries: string[] = [];
+    for (const p of result.CommonPrefixes || []) {
+      if (p.Prefix) entries.push(p.Prefix.slice(prefix.length));
+    }
+    for (const c of result.Contents || []) {
+      if (c.Key && c.Key !== prefix) entries.push(c.Key.slice(prefix.length));
+    }
+    return entries;
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)